src/routes/blog/[id$]/My Neck, My Back, My Updates and My App .md

```diff
@@ -0,0 +1,253 @@
+
+All these solutions will be available within the upcoming days/weeks \- stay tuned. 
+
+## Outline
+
+- Historical reason for CodePush and similar apps   
+- Market Comparison   
+  [https://gronxb.github.io/hot-updater/guide/getting-started/quick-start-with-supabase.html](https://gronxb.github.io/hot-updater/guide/getting-started/quick-start-with-supabase.html) (Denis think it might be good?)  
+  [https://github.com/gronxb/hot-updater](https://github.com/gronxb/hot-updater)  
+  [https://github.com/vantuan88291/react-native-ota-hot-update](https://github.com/vantuan88291/react-native-ota-hot-update)  
+  [https://ethern.dev/](https://ethern.dev/)   
+  [https://docs.expo.dev/versions/latest/sdk/updates/](https://docs.expo.dev/versions/latest/sdk/updates/)   
+  [https://rechunk.xyz/](https://rechunk.xyz/)   
+  [https://github.com/crherman7/metro-requirex](https://github.com/crherman7/metro-requirex)   
+  [https://github.com/crherman7/rechunk](https://github.com/crherman7/rechunk)   
+- How people are using CodePush and how is it enabled   
+- The Zephyr Way   
+  You will see us in the recent future   
+- Reference   
+  [https://www.techtarget.com/searchmobilecomputing/definition/OTA-update-over-the-air-update](https://www.techtarget.com/searchmobilecomputing/definition/OTA-update-over-the-air-update)   
+  [https://en.wikipedia.org/wiki/Over-the-air\_update](https://en.wikipedia.org/wiki/Over-the-air_update)   
+  [https://carbuzz.com/the-history-of-over-the-air-updates/](https://carbuzz.com/the-history-of-over-the-air-updates/)   
+  [https://theapplewiki.com/wiki/OTA\_Updates](https://theapplewiki.com/wiki/OTA_Updates)   
+  [https://source.android.com/docs/core/ota](https://source.android.com/docs/core/ota)   
+  [https://www.sonatus.com/blog/what-is-ota-a-comprehensive-guide/](https://www.sonatus.com/blog/what-is-ota-a-comprehensive-guide/)   
+  [https://www.ezurio.com/resources/blog/ota-over-the-air-updates-for-iot-devices](https://www.ezurio.com/resources/blog/ota-over-the-air-updates-for-iot-devices)   
+  [https://www.freertos.org/Documentation/03-Libraries/07-Modular-over-the-air-updates/01-Over-the-air-updates](https://www.freertos.org/Documentation/03-Libraries/07-Modular-over-the-air-updates/01-Over-the-air-updates)   
+  Why people start to use CodePush (inspiration, won’t be included)  
+- Want to deliver hot fixes (Small feature or hot fixes \- as long as there is no native dependencies)   
+- It was part of AppCenter (where apps are getting tested, submit to App Store but with limited feature)  
+- CI/CD might be on Azure/AWS   
+- Started the project with react-native-community-cli   
+  With CodePush \- to have access to releases, the application need to have pre-generated IDs, all builds will send the pre-generated ID, when you upload/deploy a MiniApp, it will also have a pre-generated ID and the app (local device on user end) also need to be aware what this ID is.   
```

Comment:
```suggestion
  With CodePush \- to have access to releases, the application needs to have pre-generated IDs, all builds will send the pre-generated ID, when you upload/deploy a MiniApp, it will also have a pre-generated ID and the app (local device on user end) also need to be aware what this ID is.   
```